scripts/service-startup-check.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+# zenoss-inspector-info
+# zenoss-inspector-tags os verify
```

Comment:
Do we need the `os` tag here? if so, it needs to be added to the `serviced-service-list-v.sh` script also.